build/lib/src/analyzer/resolver.dart

```diff
@@ -18,10 +18,15 @@ abstract class Resolver {
   /// or is a `part of` file (not a standalone Dart library).
   Future<bool> isLibrary(AssetId assetId);
 
-  /// All libraries recursively accessible from the entry point or subsequent
-  /// calls to [libraryFor] and [isLibrary].
+  /// All libraries resolved by this resolver.
```

Comment:
I have updated the documentation of this getter and `findLibraryByName` to be consistent with the current implementation in `build_resolvers`.

In that implementation, neither `isLibrary` nor `compilationUnitFor` add an entrypoint. That behavior feels correct to me (and I'm not aware of any complaints about this), so I'm in favor of updating the documentation. I've also added a test for this behavior (calling `isLibrary` should not alter `libraries`).

Reply:
SGTM